mirrord/layer/src/file/ops.rs

```diff
@@ -337,6 +337,60 @@ pub(crate) fn read_link(path: Detour<PathBuf>) -> Detour<ReadLinkFileResponse> {
     }
 }
 
+#[mirrord_layer_macro::instrument(level = Level::TRACE, ret)]
+pub(crate) fn mkdir(pathname: Detour<PathBuf>, mode: u32) -> Detour<MakeDirResponse> {
+    let pathname = pathname?;
+
+    check_relative_paths!(pathname);
```

Comment:
I copied this from `open` function, and I just wanted to validate if that validation is there because mirrord is not able to get the working directory of the original process (the one that runs on the target pod)

Reply:
Yeah, getting the working directory would be problematic (what if there were multiple processes in the target container, what if workdir changes, stuff like this). Good call to include this check ^^